### PR TITLE
MOTECH-1912 Docs update

### PR DESCRIPTION
**Pull request by jablonskidawid**
update of mtraining docs file connected with MOTECH-1912 ticket

### CODE REVIEW

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

File: docs/source/mtraining.rst [hunk 1]
```diff
@@ -67,17 +67,21 @@ Quiz
...
 +----------+------------------------------------------------------------------------------------+--------------------+
-|answer    |The answer resource identifier in the external system, for example audio file.      |String              |
-|          |This field can also store the correct answer.                                       |                    |
+|answers   |The list of answers for the question                                                |List<Answer>        |
 +----------+------------------------------------------------------------------------------------+--------------------+
 
+Answer
+######
+
+``Answer`` is a resource identifier in the external system, for example audio file. It can also store the correct answer.
```
Comment (martokarski) [inline docs/source/mtraining.rst]: Better is to say that this field can store explicit value not only reference to another resource

**Comment by martokarski** [comment]
test this please

**Comment by martokarski** [comment]
@kmadej Looks good to me.

**Pull request merged**